README.rst

```diff
@@ -1 +1,16 @@
-# django-admin-api
+=========
+ADMIN-API
+=========
```

Comment:
Add the intro line from Github in here "Expose Django's admin as a RESTful service".